libs/game/scenes.ts

```diff
@@ -1,6 +1,14 @@
 /**
  * Control the background, tiles and camera
  */
+
+enum CameraProperty {
+    left,
```

Comment:
Follow typescript naming inventions (capitalized) then use % block=« ... » to make loalizable